tests/test_light.py

```
@@ -1,10 +1,12 @@
 """Provide tests for OpenEVSE light platform."""
 
-from unittest.mock import patch
+from asyncio import TimeoutError
```

Ruff (0.15.10)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)